primitives/src/traits.rs

```diff
@@ -96,6 +97,22 @@ pub trait AMM<AccountId, AssetId, AssetPair, Amount> {
 		Self::execute_buy(&Self::validate_buy(origin, assets, amount, max_limit, discount)?)?;
 		Ok(())
 	}
+
+	fn get_min_trading_limit() -> Amount {
+		Zero::zero()
+	}
+
+	fn get_min_pool_liquidity() -> Amount {
+		Zero::zero()
+	}
+
+	fn get_max_in_ratio() -> u128 {
+		Zero::zero()
+	}
+
+	fn get_max_out_ratio() -> u128 {
+		Zero::zero()
+	}
```

Comment:
Are the default implementations really necessary ?  I would think that this could lead to some issues.

Reply:
The default implementations have been removed.